# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/grub2/hotfix-auconf-archive.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- grub-2.14/configure.ac.vanilla	2026-01-20 15:36:02.882844102 +0100
+++ grub-2.14/configure.ac	2026-01-20 15:49:44.671193273 +0100
@@ -1799,7 +1799,7 @@
 
 # The error message produced by autoconf if autoconf-archive is not installed is
 # quite misleading and not very helpful. So, try point people in the right direction.
-m4_ifndef([AX_CHECK_LINK_FLAG], [m4_fatal([autoconf-archive is missing. You must install it to generate the configure script.])])
+#m4_ifndef([AX_CHECK_LINK_FLAG], [m4_fatal([autoconf-archive is missing. You must install it to generate the configure script.])])
 
 if test "x$TARGET_APPLE_LINKER" != x1 ; then
 grub_PROG_OBJCOPY_ABSOLUTE
